src/server.ts

```diff
@@ -1,30 +1,49 @@
+import cookieParser from 'cookie-parser'
+
 require('dotenv').config();
 import express from 'express'
 const mongoose = require('mongoose');
 const cors = require("cors");
 const mongoString = process.env.DATABASE_URL
+const apiRoute = require('./routes/task/task')
+const formRoute = require('./routes/user/registerData')
+const jwt = require('jsonwebtoken')
 
 const app = express()
 
 const corsOptions = {
-    origin: "http://localhost:3000"
+    origin: "http://localhost:3000",

```

Comment:
jako, że używasz już zmiennych środowiskowych to dodaj ten adres do pliku .env

Reply:
codebase nie powinien miec zaszytych adresów w kodzie bo później to komplikuje sprawę jeżeli być wystawiał środowisko produkcyjne, które wymaga dodania do corsów inny adres niż lokalny